applications/nrf_desktop/configuration/nrf54lm20dk_nrf54lm20a_cpuapp/prj_llvm.conf

```diff
@@ -97,6 +97,12 @@ CONFIG_LED_PWM=y
 CONFIG_STREAM_FLASH=y
 CONFIG_IMG_MANAGER=y
 CONFIG_MCUBOOT_IMG_MANAGER=y
+# Enable retained memory and retention to enable the Image Manager in the MCUboot RAM load mode.
+CONFIG_RETAINED_MEM=y
+CONFIG_RETENTION=y
+CONFIG_RETAINED_MEM_ZEPHYR_RAM=y
+CONFIG_RETENTION_BOOTLOADER_INFO=y
+CONFIG_RETENTION_BOOTLOADER_INFO_TYPE_MCUBOOT=y
```

Comment:
Consider explicitly enabling the `CONFIG_RETENTION_BOOTLOADER_INFO_OUTPUT_FUNCTION` (as it's dependency of `CONFIG_IMG_MANAGER`)